astacus/common/ipc.py

```diff
@@ -195,9 +195,8 @@ class SnapshotClearRequest(NodeRequest):
 class CassandraSubOp(StrEnum):
     get_schema_hash = "get-schema-hash"
     remove_snapshot = "remove-snapshot"
-    unrestore_snapshot = "unrestore-snapshot"
-    restore_snapshot = "restore-snapshot"
-    restore_snapshot_with_schema = "restore-snapshot-with-schema"
+    unrestore_sstables = "unrestore-sstables"
+    restore_sstables = "restore-sstables"
```

Comment:
N.B.: since this is restore, I don't worry about backward compatibilty, since partial restore is not yet supported -> if we restore, all the nodes are likely to be on the newest version.